donut/modules/uploads/templates/page.html

```diff
@@ -0,0 +1,19 @@
+{% extends "layout.html" %}
+{% block page %}
+<script src="https://cdn.rawgit.com/showdownjs/showdown/1.0.1/dist/showdown.min.js"></script>
+<script src="{{ url_for('uploads.static', filename='script.js', _external = True)}}" type="text/javascript"></script>
+<div class="container">
+  <div class="col-md-24">
+    <div class="jumbotron">
+      <h1 id = "title" class="text-center">{{title}}</h1>
+      <div id = "content">
+
+      <script>convert(`{{page}}`);</script>
```

Comment:
JS strings should use quotes; the backtick is only supported in modern versions of JS

Reply:
So I have a pretty ugly way around this, but honestly it would be preferable to be able to have multi line comments

Reply:
Couldn't you just have the JS make an AJAX call to a Flask endpoint that sends it the contents?